great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -153,12 +153,17 @@ def __init__(
 
         self._usage_statistics_handler = usage_statistics_handler
 
-        # Necessary to annotate ExpectationSuite during `run()`
+        # Necessary to annotation ExpectationSuite during `run()`
+        rule_to_load_into_citation: dict = {}
+        if isinstance(rules, Rule):
+            rule_to_load_into_citation = rules.to_json_dict()
+        elif isinstance(rules, dict):
+            rule_to_load_into_citation = rules
         self._citation = {
             "name": name,
             "config_version": config_version,
             "variables": variables,
-            "rules": rules,
+            "rules": rule_to_load_into_citation,
```

Comment:
@Shinnnyshinshin Based on the above comments, it seems that this can be left as it was before.  Happy to discuss. 
 Thanks.

Reply:
the `add_rule()` method and reconciliation made this part unnecessary. Thanks @alexsherstinsky